.github/workflows/testsuite.yml

```diff
@@ -0,0 +1,152 @@
+name: testsuite
+
+on:
+  push:
+    branches:
+      - "*"
+    tags-ignore:
+      - "*"
+  pull_request:
+
+jobs:
+  ubuntu:
+    env:
+      PERL_USE_UNSAFE_INC: 0
+      AUTHOR_TESTING: 1
+      AUTOMATED_TESTING: 1
+      RELEASE_TESTING: 1
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: perl -V
+        run: perl -V
+      - name: install dependencies
+        uses: perl-actions/install-with-cpm@stable
+        with:
+          install: |
+            File::Temp
```

Comment:
Shouldn't this be handled automatically? What's going on here?

Reply:
Also, this looks like a recipe for circular dependencies, this can easily lead to version being installed before we want that to happen. We may have to drop that `parent` dependency

Reply:
You are right, after some testing this is not necessary to pull/update the dependencies listed in Makefile.PL

Reply:
I've updated the workflow to avoid updating them, view 822b10f6